docs/main/index.md

```diff
@@ -1,60 +0,0 @@
-# Introduction
```

Comment:
Was this on purpose? Should we add any of the extract stuff to the docs? Like how to work with a HeroExtractor?